docs/security.md

```diff
@@ -4,36 +4,80 @@ displayTitle: Spark Security
 title: Security
 ---
 
-Spark currently supports authentication via a shared secret. Authentication can be configured to be on via the `spark.authenticate` configuration parameter. This parameter controls whether the Spark communication protocols do authentication using the shared secret. This authentication is a basic handshake to make sure both sides have the same shared secret and are allowed to communicate. If the shared secret is not identical they will not be allowed to communicate. The shared secret is created as follows:
-
-* For Spark on [YARN](running-on-yarn.html) deployments, configuring `spark.authenticate` to `true` will automatically handle generating and distributing the shared secret. Each application will use a unique shared secret. 
-* For other types of Spark deployments, the Spark parameter `spark.authenticate.secret` should be configured on each of the nodes. This secret will be used by all the Master/Workers and applications.
+Spark currently supports authentication via a shared secret. Authentication can be configured to be
```

Comment:
I meant just to break the long lines you were modifying, not all of them. :-( Sorry if I wasn't clear. Not a big deal IMO if it's too annoying to revert.